mlir/include/Quantum/IR/QuantumOps.td

```diff
@@ -522,7 +522,8 @@ def MeasureOp : Quantum_Op<"measure"> {
     }];
 
     let arguments = (ins
-        QubitType:$in_qubit
+        QubitType:$in_qubit,
+        OptionalAttr<ConfinedAttr<I8Attr, [IntMinValue<-1>, IntMaxValue<1>]>>:$postselect
```

Comment:
After your recent commit the minimum can now be changed :)
```suggestion
        OptionalAttr<ConfinedAttr<I8Attr, [IntMinValue<0>, IntMaxValue<1>]>>:$postselect
```

Reply:
True, I didn't realize. Done.